examples/docker/modin-hdk/nyc-taxi-hdk.py

```diff
@@ -179,25 +220,66 @@ def q4_hdk(df):
         .size()
         .reset_index()
         .sort_values(
-            by=["pickup_datetime", 0], ignore_index=True, ascending=[True, False]
+            by=["pickup_datetime", 0, "passenger_count", "trip_distance"],
```

Comment:
Why have the new values been added for the parameters?

Reply:
Just because the row order is a bit different between sort_values() and sql, unless the order is specified explicitly.

Reply:
We should have these examples aligned with the omniscripts. Do you plan to fix the examples in omniscripts as well?

Reply:
@Garra1980

Reply:
Replaced with additional sorting to the results.